force-app/main/default/aura/disbursementsCreate/disbursementsCreate.cmp

```diff
@@ -191,20 +219,23 @@
         <aura:if isTrue="{!not(empty(v.model.disbursements))}">
             <div class="slds-section slds-is-open">
                 <div class="slds-section__content">
-                    <lightning:datatable
-                        keyField="id"
-                        data="{!v.model.disbursements}"
+                    <c:dataTableWrapper
+                        key-field="id"
+                        records="{!v.model.disbursements}"
                         columns="{!v.model.formDefaults.columns}"
-                        hideCheckboxColumn="true"
-                        oncellchange="{!c.handleRowChange}"
-                        aura:id="disbursementTable"
+                        ondratfvaluechange="{!c.handleLWCRowChange}"
```

Comment:
Looks like a typo here, should be  `draftvaluechange` instead of `dratfvaluechange`

Reply:
@seanpat09  I just pushed an update for these typos.